src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -124,9 +124,21 @@
   </ItemGroup>
   <!-- SocketsHttpHandler implementation -->
   <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
+    <Compile Include="System\Net\Http\HPack\DynamicTable.cs" />
+    <Compile Include="System\Net\Http\HPack\HeaderField.cs" />
+    <Compile Include="System\Net\Http\HPack\HPackDecoder.cs" />
+    <Compile Include="System\Net\Http\HPack\HPackDecodingException.cs" />
+    <Compile Include="System\Net\Http\HPack\HPackEncoder.cs" />
+    <Compile Include="System\Net\Http\HPack\Huffman.cs" />
+    <Compile Include="System\Net\Http\HPack\HuffmanDecodingException.cs" />
+    <Compile Include="System\Net\Http\HPack\IntegerDecoder.cs" />
+    <Compile Include="System\Net\Http\HPack\IntegerEncoder.cs" />
+    <Compile Include="System\Net\Http\HPack\StaticTable.cs" />
```

Comment:
Were you going to move this HPack directory to be under SocketsHttpHandler?

Reply:
I'll do that in a subsequent checkin.